content/200-orm/200-prisma-client/100-queries/060-full-text-search.mdx

```diff
@@ -247,14 +247,15 @@ However, if you try to search on `title` alone, the search will fail with the er
 
 ## Full-text search with raw SQL
 
-Full-text search is currently in Preview and due to a [known issue](https://github.com/prisma/prisma/issues/23627), you may be seeing slow search queries. If that's the case, you can optimize your query using [raw SQL](/orm/prisma-client/queries/raw-database-access).
+Full-text search is currently in Preview and due to a [known issue](https://github.com/prisma/prisma/issues/23627), you may be seeing slow search queries. If that's the case, you can optimize your query using [TypedSQL](/orm/prisma-client/using-raw-sql).
 
 ### PostgreSQL
 
 In PostgreSQL, you can use `to_tsvector` and `to_tsquery` to express your search query:
 
 ```ts
 const term = `cat`
+// TODO
```

Comment:
@SevInf is this a query that could be in TypedSQL now? I wasn't sure since it was using `tsvector` and `tsquery` but it looks like the resulting columns don't contain unsupported features.

Reply:
yes, i think it can